### ISSUE TITLE
Feature request: Publishing on F-Droid

### ISSUE THREAD

**Issue by marek22k** [MatsuriDayo/NekoBoxForAndroid#14]
F-Droid is a well-known appstore that offers FLOSS apps only, without any proprietary dependencies. As far as I can see, this app is completely open-source/free. Maybe it would be possible to publish it on F-Droid?

**Comment by linsui** [comment]
I thought we should also mention this app in the description of matsuri.

@arm64v8a Can I package this? Is NekoBox ready?

**Comment by tdjnodj** [comment]
Nekobox for Android is selling in the play store.Do you know what's that mean?

**Comment by marek22k** [comment]
> Nekobox for Android is selling in the play store.Do you know what's that mean?

Well, in an alternative F-Droid repo you can already download the app: https://apt.izzysoft.de/fdroid/index/apk/moe.nb4a

But it would be nothing special that apps are free in F-Droid and cost something in the Google App Store (for example the Simple series or the Termux extensions).

**Comment by tdjnodj** [comment]
@marek22k Good job!

**Comment by marek22k** [comment]
I had nothing to do with it. I just thought that if the app is already available on Izzy, it would be even nicer if it is available in F-Droid.

**Comment by linsui** [comment]
It's available because it's available in GitHub release for free.

**Comment by linsui** [comment]
Please mention that Matsuri is discontinued in its description.

**Comment by marek22k** [comment]
> I don't want to put Nekobox on F-Droid for now. @linsui @marek22k

In this case, maybe close as not planned.

**Comment by linsui** [comment]
What update do you want? NekoBox won't be published on F-Droid.

**Comment by marek22k** [comment]
> In addition, I may release a "final version" for Matsuri if it has nothing to update, which can be sync to F-Droid.

@linsui Mhh, apparently there is the possibility to bring an app version in F-Droid.

**Comment by linsui** [comment]
@marek22k It's about Matsuri, not NekoBox.

**Comment by OmlineEditor** [comment]
can someone else add this program to the android catalog? Does the developer agree or not? the developer does not need to do anything, other people will do everything. The developer should only give consent and permission to other people to add the program.

**Comment by linsui** [comment]
The dev said:

> I don't want to put Nekobox on F-Droid for now.

**Comment by OmlineEditor** [comment]
> The dev said:
> 
> > I don't want to put Nekobox on F-Droid for now.

why doesn't he want to? If another person does this, then nothing will be required from him

**Comment by linsui** [comment]
I don't know. But it seems the original dev has left this project so you should ask the current maintainers instead.

**Comment by ahangarha** [comment]
If they don't want to do so, I would propose that someone who knows the process, make a fork for adding the app to f-droid.

The only reasonable blocker is that they don't know what to do and don't want to spend time on it. If it is so, they can communicate with others and ask for contributions. Even better, send a PR and wait for approval or rejection. Then decide about making a fork or not.

**Comment by linsui** [comment]
> The only reasonable blocker is that they don't know what to do and don't want to spend time on it.

No. They know how to do that and they did that for Matsuri. There is another sing-box client on F-Droid if that's what you want.

**Comment by OmlineEditor** [comment]
You don't need to do anything to work with F-Droid, someone else can do it. Then you don't need to do anything to add new versions, it will be done automatically. I'm just trying to ask **_only for permission_** to make an addition on the F-Droid.

**Comment by Integral-Tech** [comment]
Any progress on this issue?

**Comment by Integral-Tech** [comment]
> It's available in IzzySoft's repo because it's available in GitHub release for free.

However NekoBox in IzzyOnDroid is armeabi-v7a, not arm64-v8a

**Comment by linsui** [comment]
Please stop bumping this **closed** issue. :)

**Comment by OmlineEditor** [comment]
Why was the ticket closed? If the problem is not solved, please open it again

**Comment by Korb** [comment]
> Why was the ticket closed? If the problem is not solved, please open it again

The problem has been solved, the application is present in F-Droid, everyone can find it in the application store, provided that the corresponding repository is connected, and enjoy the benefits of automatic updating:

![Screenshot_2024_1212_123652_org fdroid fdroid](https://github.com/user-attachments/assets/22bb87c5-6e23-4eb9-b727-6e1fb35308aa)

**Comment by marek22k** [comment]
I guess it depends on how you look at it. It is available in the IzzyOnDroid F-Droid repo, but not in the official F-Droid repo.

**Comment by Integral-Tech** [comment]
However, IzzyOnDroid F-Droid repo only provides 32-bit armeabi-v7a builds
